pkg/util/flagext/stringslice.go

```diff
@@ -1,15 +1,13 @@
 package flagext
 
-import (
-	"strings"
-)
+import "fmt"
 
 // StringSlice is a slice of strings that implements flag.Value
 type StringSlice []string
 
 // String implements flag.Value
 func (v StringSlice) String() string {
-	return strings.Join(v, " ")
+	return fmt.Sprintf("%s", []string(v))
```

Comment:
The unification of the flag extensions is basically that I kept the name `StringSlice` but "borrowed" the `String` casting function.

I like this because for empty values it renders `[]` instead of `""` which seems to be more accurate.